src/data/reducers/connectionStatus.js

```diff
@@ -0,0 +1,10 @@
+import { PING_RESPONSE } from '../actions/pingStudio';
+
+export default function connectionStatus(state = null, action) {
```

Comment:
@dsjen `export default` works inline here because `connectionStatus` is a named function declared as a `FunctionDeclaration`, whereas `WrappedBackendStatusBanner` is declared as a LexicalDeclaration. Basically, if it contains an = sign or could be assigned to a variable, it's a LexicalDeclaration and has to be exported default separately. If it can't be assigned to a variable (e.g. `class MyClass {`, `function myFunction`, etc) it's a HoistableDeclaration, ClassDeclaration or AssignmentExpression, and we can export default it inline.

@MichaelRoytman that's a good point. Now, if we change this function to an arrow function instead of a named function, we end up with something like `const connectionStatus = () => {`. That turns it into a LexicalDeclaration, and we'll have to add an `export default connectionStatus;` at the bottom of the file like with the `WrappedBackendStatusBanner`. I'm going to make that change, at least for consistency's sake -- as long as we're cool with `export default`s at the bottom of files. (I am!)

Reply:
Sounds like a plan! I appreciate you taking the time to explain what's going on here, @arizzitano.